packages/cli-kit/src/private/node/ui/components/TextAnimation.tsx

```diff
@@ -37,7 +37,7 @@ const TextAnimation: React.FC<Props> = ({text}): JSX.Element => {
     renderAnimation()
 
     return () => {
-      clearTimeout(timeout.current)
+      clearTimeout(timeout.current as unknown as number)
```

Comment:
Curious what changed

Reply:
And same here,  the version `@types/node` that we are using now has the types for timeouts wrongly defined so I'm doing type erasing.